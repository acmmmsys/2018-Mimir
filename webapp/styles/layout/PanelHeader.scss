@import '../colors';

.panel-header {
    display: flex!important;
    flex-direction: row!important;
    justify-content: flex-end!important;
    align-items: center!important;
    flex-wrap: nowrap;
    
    border-left: 2px solid lighten($backgroundColor, 5%);
    border-right: 2px solid darken($backgroundColor, 5%);
    border-top: 2px solid lighten($backgroundColor, 5%);
    border-bottom: 2px solid darken($backgroundColor, 5%);
}