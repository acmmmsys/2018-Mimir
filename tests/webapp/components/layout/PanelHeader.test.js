import React from 'react';
import PanelHeader from '../../../../webapp/components/layout/PanelHeader'
import renderer from 'react-test-renderer';

let wrapper;

test('PanelHeader does render', () => {
    wrapper = shallow(
        <PanelHeader>

        </PanelHeader>
    );
    expect(wrapper).toMatchSnapshot();
});
