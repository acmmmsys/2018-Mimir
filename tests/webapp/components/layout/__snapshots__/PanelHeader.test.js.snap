// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelHeader does render 1`] = `
<branch(branch(PanelHeader))
  emptyMessage=""
  loading={false}
/>
`;
